clash-cores/src/Clash/Cores/I2C/ByteMaster.hs

```diff
@@ -29,10 +29,8 @@ data ByteMasterS
   , _coreTxd    :: Bit              -- coreTxd register
   , _shiftsr    :: Bool             -- shift sr
   , _ld         :: Bool             -- load values in to sr
-  , _i2cOpAck    :: Bool             -- host cmd acknowlegde register
```

Comment:
Why is the command ack made combinational? The commit message doesn't state why this change was made.